serde-tests/Cargo.toml

```diff
@@ -6,17 +6,17 @@ build = "build.rs"
 
 
 [dependencies]
-serde = "0.7"
+serde = "0.8"
 bson = { path = "..", features = ["serde"] }
 
 [build-dependencies]
-syntex = "^0.29.0"
-serde_codegen = "0.7"
+syntex = "*"
```

Comment:
Mmm... this is in the Cargo.toml from serde-tests, which is not meant to be deployed to crates, but I can try and fix the version anyway.